Feat: Added video, mention of and link to Yarn 6 Playground (#453)
leaningtech/labs: sites/labs/src/content/blog/browserpod-rust.mdx

Reviewer: ```suggestion
## What is BrowserPod?
```

Reviewer: ```suggestion
## What does it mean to "support" Rust?
```

	userImage="wss://disks.browserpod.io/rust-post-demos-2.ext2"
	apiKey="bp1_2cc277b71b4bdd4850b0ee9bd117b2354e9be99d6a5b1cf3379f8135c84a8a5a"
/>

> Today we are releasing BrowserPod 3.0, with full support for running any Rust

Reviewer: ```suggestion
Today we are releasing [BrowserPod 3.0](https://browserpod.io/), with full support for running any Rust
```

Reviewer: I noticed there is no link to browserpod until the end of the post

Programs can access the filesystem, make network requests, run subprocesses, and interact
with concurrently running applications. All of this **without changing a single line of code**!

> In the video below, you can see [Yarn 6](https://v6.yarnpkg.com) (written in Rust) running in Yarn's [official v6 Playground](https://v6.yarnpkg.com/playground):

Reviewer: ```suggestion
In the video below, you can see [Yarn 6](https://v6.yarnpkg.com) (written in Rust) running in Yarn's [official v6 Playground](https://v6.yarnpkg.com/playground): [^1]
```

<video controls autoplay loop muted playsinline>
	<source src="/blog/bp-rust/yarn-playground.mp4" type="video/mp4" />
</video>

> You can test Yarn in the below Terminal, or try it out in the [official Yarn 6 Playground](https://v6.yarnpkg.com/playground).

Reviewer 2: ```suggestion
You can also test Yarn in the Terminal below if you prefer
```

Playground link is redundant

Author: Good shout. I've updated this copy now to remove the excessive linking.

Head to [console.browserpod.io](https://console.browserpod.io) to get an API key. You can find the documentation on [browserpod.io/docs](https://browserpod.io/docs).

If you have a Rust project you'd like to see running in a browser tab (especially one you expect to break) we'd genuinely like to hear about it. Come find us on [Discord](https://discord.leaningtech.com).

> - This blog was amended on 17 August 2026 to mention Yarn 6 embedding BrowserPod in its official playground.

Reviewer: ```suggestion
[^1]: This blog was amended on 17 August 2026 to mention Yarn 6 embedding BrowserPod in its official playground.
```
